repondeur/zam_repondeur/services/fetch/an/amendements.py

```diff
@@ -57,10 +63,27 @@ def __init__(self, organe: str) -> None:
         self.organe = organe
 
 
+class ProgressBar:
```

Comment:
I wonder if we should/could reuse the [ProgressBar from progressist](https://github.com/pyrates/progressist/blob/master/progressist/__init__.py#L47) (we already have that dependency) 🤔

Reply:
No idea